index.js

```diff
@@ -46,77 +46,77 @@ async function run() {
 
   const repoFullName = github.context.payload.repository.full_name;
   const repoInLicenseMap = repoFullName in licenseMap;
-  var commit_authors = []
-  for (const i in commits.data) {
+  var commit_authors = {};
+  for (const commitObj of commits.data) {
     // Check if the commit message contains a license header that matches
     // one of the licenses granting implicit CLA approval
-    if (commits.data[i]['commit']['message']) {
+    if (commitObj.commit.message) {
       if (repoInLicenseMap) {
-        const goodLicense = hasImplicitLicense(commits.data[i]['commit']['message'], repoFullName);
+        const goodLicense = hasImplicitLicense(commitObj.commit.message, repoFullName);
         if (goodLicense) {
-          console.log('- commit ' + commits.data[i]['sha'] + ' ✓ (' + goodLicense + ' license)');
+          console.log(`- commit ${commitObj.sha} ✓ (${goodLicense} license)`);
           continue;
         }
       }
     }
 
-    var username;
-    if (commits.data[i]['author']) {
```

Comment:
by this we allowed username to stay undefined and override multiple users in the next assignment